### PR TITLE
20.11.1-rns

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.11.1.md [hunk 1]
```diff
@@ -0,0 +1,1116 @@
+# Cortex XSOAR Content Release Notes for version 20.11.1 (196572)
+##### Published on 22 November 2020
+
+### Breaking Changes
+Several packs include breaking changes.
+
+- [CrowdStrike Malquery Pack v1.0.1](#crowdstrike-malquer-pack-v101)
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.11.1.md:7]: typo in the link. It doesn't work.

File: content-repo/extra-docs/releases/20.11.1.md [hunk 2]
```diff
@@ -0,0 +1,1116 @@
...
+- [PAN-OS Pack v1.6.6](#pan-os-pack-v166)
+- [PassiveTotal Pack v2.0.5](#passivetotal-pack-v205-partner-supported)
+
+### New: Asset Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.11.1.md:11]: Let's add description what this pack is. It is not clear. Add the following:

Base pack with incident fields for any packs requiring asset fields.

File: content-repo/extra-docs/releases/20.11.1.md [hunk 3]
```diff
@@ -0,0 +1,1116 @@
...
+The MAC address provided for the asset.
+- **Switch ID - Asset**
+The ID of the switch the the asset is connected to.
+- **Switch Port ID - Asset**
+The ID of the switch port for which the asset is connected to.
+
+
+---
+
+### New: Cisco ESA IronPort Email API Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.11.1.md:33]: Add community contributed. @yaakovi aren't we adding `(Community Contributed)` automatically?
```suggestion
### New: Cisco ESA IronPort Email API Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.11.1.md [hunk 4]
```diff
@@ -0,0 +1,1116 @@
...
+#### Integrations
+##### Cisco IronPort EMail API
+Cisco ESA IronPort EMail API.
+
+
+---
+
+### New: CyberX - Central Manager Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.11.1.md:41]:
```suggestion
### New: CyberX - Central Manager Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.11.1.md [hunk 5]
```diff
@@ -0,0 +1,1116 @@
...
+
+### New: Google Calendar Pack v1.0.0
+#### Integrations
+##### Google Calendar
+Google Calendar is a time-management and scheduling calendar service developed by Google. This integration helps you to perform various tasks on the access control list (ACL).
+
+
+---
+
+### New: Palo Alto Networks Enterprise DLP Contribution Pack Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.11.1.md:84]:
```suggestion
### New: Palo Alto Networks Enterprise DLP Pack v1.0.0
```

File: content-repo/extra-docs/releases/20.11.1.md [hunk 6]
```diff
@@ -0,0 +1,1116 @@
...
+##### Rundeck
+Rundeck is a runbook automation for incident management, business continuity, and self-service operations. |- The integration enables you to install software on a list of machines or perform a task periodically. Can be used when there is a new attack and you want to perform an update of the software to block the attack.
+#### Playbooks
+##### Rundeck-job-execute-Generic
+This playbook executes a job and exits when it successfully finishes.
+
+
+---
+
+### New: Viper Pack v1.0.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.11.1.md:114]:
```suggestion
### New: Viper Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.11.1.md [hunk 7]
```diff
@@ -0,0 +1,1116 @@
...
+
+##### New: Carbon Black Live Response - Wait Until Command Complete
+Polling command status until finished with an error or complete.
+
+##### New: Carbon Black Live Response - Create active session
+Create an active session. If the active session already exists, will use the existing session.
+
+---
+
+### Chronicle Pack v1.1.3 (Partner Supported)
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.11.1.md:268]: Need to fix formatting. It has extra bullets:

image

**Comment by ShirleyDenkberg** [comment]
@glicht Doc review completed.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**